src/v2/Apps/Fair/Utils/CheckSort.ts

```diff
@@ -0,0 +1,6 @@
+export const supportedSorts = ["FEATURED_DESC", "NAME_ASC"]
+export const defaultSort = supportedSorts[0]
+
+export const checkSort = (sort: string = "") => {
```

Comment:
Maybe it would be better to rename it to `checkIsAllowedSortValue`?

Reply:
@dimatretyak yeah, I agree it's better to give the function a clearer name but I'd rather name it `isSortValid`. What do you think?

Reply:
Yes, this name is even better